src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj

```diff
@@ -15,8 +15,12 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU'" />
   <ItemGroup>
+    <ProjectReference Include="..\..\System.Security.Cryptography.Algorithms\src\System.Security.Cryptography.Algorithms.csproj">
```

Comment:
The P2P reference should be able to be deleted with the contract changes already submitted, right?